matter/wifi/wf200/host_if.cpp

```diff
@@ -580,7 +580,7 @@ static void wfx_events_task(void *p_arg)
       sl_wfx_ssid_def_t ssid, *sp;
       uint16_t num_ssid, slen;
       if (scan_ssid) {
-        memset(&ssid, 0, sizeof(ssid));
+        memset(&ssid, EMPTY_BUFFER, sizeof(ssid));
```

Comment:
Naming should match the defines used for the rs911x source files.

Reply:
changed to CLEAR_BUFFER